google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -56,6 +56,9 @@ public StorageClass apply(String constant) {
   /** Standard storage class. */
   public static final StorageClass STANDARD = type.createAndRegister("STANDARD");
 
+  /** Archive storage class. */
```

Comment:
Regional and Multi-regional are also considered deprecated now. Let's add that to the comment for each of those and also add a link to https://cloud.google.com/storage/docs/storage-classes which has the most up-to-date information.

Reply:
done